server/routes/api/choropleth.py

```diff
@@ -174,10 +173,11 @@ def choropleth_geo():
             # Process Statistical Observation if valid.
             if ('data' in population_by_geo.get(geo_id, [])
                     and population_by_geo[geo_id]['data']):
-                # Grab the latest available data.
-                latest_data = get_latest_data(population_by_geo[geo_id])
-                if latest_data:
-                    geo_feature["properties"]["pop"] = latest_data
+                # Grab the data.
+                data = get_data(population_by_geo[geo_id])
+                if data:
+                    max_date = max(data)
+                    geo_feature["properties"]["pop"] = data[max_date]
```

Comment:
I am not following the logic.  I thought the Python code would now return either the data for the desired date xor data for all dates.  This seems to return the latest date only.  What am I missing?

Reply:
The python code returns the entire time series except for population.
We only return the latest population, for now.

**data -> return entire time series data.
population -> only returns the latest dataset.**

Front-end expects a single integer for population, not a time series so I have to calculate the max_date before returning.

Reply:
Added a TODO.